Add some repro notes for benchmark failures. (#7933)
google/iree: benchmarks/README.md

### Other project metrics

TODO(#6161): Collect metrics for miscellaneous IREE system states

## Developer notes

These are ad-hoc notes added for developers to help triage errors.

> ### Repro of TFLite model errors.

Reviewer: nit: remove period from heading

```suggestion
### Repro of TFLite model errors
```

https://developers.google.com/style/periods#periods_with_headings

These steps help reproduce the failures in TFLite models. 

1.  Install `iree-import-tflite`.

```
$ python -m pip install iree-tools-tflite-snapshot -f https://github.com/google/iree/releases
```

2. Confirm the binary `iree-import-tflite` is in your path by running
```
$ iree-import-tflite --help

> ```

Reviewer 2: `````suggestion
1.  Install `iree-import-tflite`.

    ```
    $ python -m pip install iree-tools-tflite-snapshot -f https://github.com/google/iree/releases
    ```

2. Confirm the binary `iree-import-tflite` is in your path by running
    ```
    $ iree-import-tflite --help
    ```
`````

> 3. Download the TFLite flatbuffer for the failing benchmarks. The
> ocation can be found from [this CMakeLists.txt file](https://github.com/google/iree/blob/f1678fb282d561fbb771d9ebfc97d4a3cdfa3fd3/benchmarks/TFLite/CMakeLists.txt)

Reviewer: you can just use a relative path here, without pinning to a specific git commit

```suggestion
3. Download the TFLite flatbuffer for the failing benchmarks. The
location can be found from [this CMakeLists.txt file](./TFLite/CMakeLists.txt)
```

> 4. The input TOSA model can be generated by running
> ```
> $ iree-import-tflite <tflite file> -o <tosa output file>
> ```

Reviewer 2: `````suggestion
4. The input TOSA model can be generated by running
    ```
    $ iree-import-tflite <tflite file> -o <tosa output file>
    ```
`````

> 5. The exact flags used to compiler and run the benchmarks can be
> found in [this CMakeLists.txt file](https://github.com/google/iree/blob/f1678fb282d561fbb771d9ebfc97d4a3cdfa3fd3/benchmarks/TFLite/CMakeLists.txt)v

Reviewer: same down here

```suggestion
5. The exact flags used to compiler and run the benchmarks can be
found in [this CMakeLists.txt file](./TFLite/CMakeLists.txt)v
```
